js/components/skipControls.js

```diff
@@ -244,9 +276,15 @@ class SkipControls extends React.Component {
    */
   getSortedButtonEntries() {
     const buttons = [];
+    let key;
+    if (this.props.audioOnly) {
```

Comment:
Would it be better to pass the right button configuration as a prop instead of checking for audio only? The component could just use that object instead of looking for it directly on `skinConfig`

Reply:
I guess I wanted to try to keep the configurability of the skip controls. I'll double check with Steve, but if we don't want this to be configurable, I'll make this change.